packages/compass-settings/src/components/modal.tsx

```diff
@@ -95,7 +101,8 @@ export const SettingsModal: React.FunctionComponent<SettingsModalProps> = ({
           className={settingsStyles}
           data-testid="settings-modal-content"
           role="tabpanel"
-          id={`tabpanel-${selectedSetting}`}
+          tabIndex={0}
```

Comment:
It is suggested that tabpanels where first element is not focusable (like a link) are focusable, [see example](https://www.w3.org/WAI/ARIA/apg/example-index/tabs/tabs-manual.html)